dependencies/CMakeLists.txt

```diff
@@ -2,6 +2,14 @@
 
 set(current_dir ${CMAKE_CURRENT_SOURCE_DIR})
 
+add_subdirectory(${current_dir}/glm)
+add_subdirectory(${current_dir}/sdl)
+add_subdirectory(${current_dir}/plog)
+set(OPTION_BUILD_EXAMPLES OFF CACHE BOOL "Build examples." FORCE)
+set(BUILD_SHARED_LIBS OFF CACHE BOOL "Build shared instead of static libraries." FORCE)
```

Comment:
sadly we need to set this cmake options to build our glbinding lib correctly
and we should decide whether we want every dependency to be static to our project
if that's the case - we should duplicate 9-th before every `add_subdirectory()` call :)